src/pages/HomePage/Componets/OurServices/OurServicesListItem/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import "./OurServicesListItem.css";
+
+const OurServicesListItem = ({ serviceItem }) => {
```

Comment:
Nice job! Passing `serviceItem` as a whole simplifies things and makes them less abstract. Love it 👍